### ISSUE TITLE
Using invalid properties in security scheme declaration

### ISSUE THREAD

**Issue by postatum** [aml-org/amf#496]
### Issue
Running code below should fail or report validation error because according to RAML 1.0 spec provided RAML input is invalid.

The spec says:
> The security scheme node is a map that has the following key-value pairs: type, displayName?, description?, describedBy?, settings?.

Spec link:
https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#security-scheme-declaration

### AMF Version
4.0.0-SNAPSHOT

### Code
```sh
$ java -jar amf-4.0.0-SNAPSHOT.jar validate -in "RAML 1.0" -mime-in "application/yaml" -p "RAML" file://api.raml
{
  "@type": "http://www.w3.org/ns/shacl#ValidationReport",
  "http://www.w3.org/ns/shacl#conforms": true
}
```

### api.raml
This issue is also true when invalid properties are defined in `#%RAML 1.0 SecurityScheme` fragment.

```raml
#%RAML 1.0
title: Demo
version: v1
baseUri: http://api.samplehost.com

securitySchemes:
  oauth2:
    invalidProperty: 1
    type: OAuth 2.0
    settings:
      authorizationUri: http://example.com
      accessTokenUri: http://example.com
      authorizationGrants: http://example.com
      scopes: http://example.com

/resource:
  get:
    securedBy: [ null, oauth2 ]
```

**Comment by gutee** [comment]
Jira issue to track this: https://www.mulesoft.org/jira/browse/APIMF-1522

**Comment by gutee** [comment]
This is fixed and stated for 3.3.0 release
